.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+on:
+  push:
+    branches:
+      - master 
+  pull_request:
```

Comment:
We need to run the CI on pull requests that target `master` as well.

```
 pull_request:
   branches: 
      -  master
```

Reply:
I think it's the default, but I will add it just in case

Reply:
Added!

Reply:
okay I thought it was missing and that was why the CI was not running